pom.xml

```diff
@@ -282,10 +279,10 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.1</version>
+        <version>3.7.0</version>
```

Comment:
You forgot to change
`<_exportcontents>co.cask.hydrator.*</_exportcontents>`

Reply:
Done.